collect_env.py

```diff
@@ -519,6 +522,16 @@ def is_xnnpack_available():
     else:
         return "N/A"
 
+def get_env_vars():
+    env_vars = ''
+    BLACK_LIST=('S3_SECRET_ACCESS_KEY', 'VLLM_API_KEY', 'S3_ACCESS_KEY_ID')
+    for k, v in os.environ.items():
+        if k in BLACK_LIST:
+            continue
```

Comment:
We could use `v = ***` or something like that to denote that the environment variable was set, but redacted here

Reply:
Good point. Done